doc/classes/@GlobalScope.xml

```diff
@@ -53,7 +53,7 @@
 			The [JavaScript] singleton.
 			[b]Note:[/b] Only implemented on HTML5.
 		</member>
-		<member name="Marshalls" type="Reference" setter="" getter="">
+		<member name="Marshalls" type="Object" setter="" getter="">
```

Comment:
Shouldn't it be of type `Marshalls`?

Reply:
I just checked by running `godot --doctool .` and indeed it properly infers the type as `Marshalls`.

Reply:
Yes, you're right. I just followed the logic. I force pushed the change because it's very simple stuff